docs/pages/rigid-object-tutorial.rst

```diff
@@ -0,0 +1,196 @@
+    :code: py
+    :start-after: # [kinematic_interactions]
+    :end-before: # [/kinematic_interactions]
+
+.. image:: images/rigid-object-tutorial-images/kinematic_interactions.gif
+    :width: 20em
+
+
+`Kinematic Velocity Control`_
+=============================
+
+
+To move a kinematic object, the state can be set directly before each simulation step.
+This is useful for synchronizing the simulation state of objects to a known state such as a dataset trajectory, input device, or motion capture.
+
+.. include:: ../../examples/tutorials/rigid_object_tutorial.py
+    :code: py
+    :start-after: # [kinematic_update]
+    :end-before: # [/kinematic_update]
+
+.. image:: images/rigid-object-tutorial-images/kinematic_update.gif
+    :width: 20em
+
+However, when applying model or algorithmic control it is more convenient to specify a constant linear and angular velocity for the object which will be simulated without manual integration.
+The object's :ref:`habitat_sim.physics.VelocityControl` structure provides this functionality and can be acquired via :ref:`Simulator.get_object_velocity_control`.
+Once paramters are set, control takes affect immediately on the next simulation step as shown in the following example.
+
+.. include:: ../../examples/tutorials/rigid_object_tutorial.py
+    :code: py
+    :start-after: # [velocity_control]
+    :end-before: # [/velocity_control]
+
+.. image:: images/rigid-object-tutorial-images/velocity_control.gif
+    :width: 20em
+
+Velocities can also be specified in the local space of the object to easily apply velocity control for continuous agent actions.
+
+.. include:: ../../examples/tutorials/rigid_object_tutorial.py
+    :code: py
+    :start-after: # [local_velocity_control]
+    :end-before: # [/local_velocity_control]
+
+.. image:: images/rigid-object-tutorial-images/local_velocity_control.gif
+    :width: 20em
+
+`Embodied Agents`_
+==================
+
+For this tutorial section, you will need to download the `merged locobot asset`_ and extract it into path/to/habitat-sim/data/objects/
+
+.. _merged locobot asset: http://dl.fbaipublicfiles.com/habitat/locobot_merged.zip
+
+Previous stages of this tutorial have covered adding objects to the world and manipulating them by setting positions, velocity, forces, and torques.
+In all of these examples, the agent has been a passive onlooker observing the scene.
+However, the agent can also be attached to a simulated object for embodiement and control.
+This can be done by passing the :ref:`Agent`'s scene node to the :ref:`Simulator.add_object` function.
+
+In this example, the agent is embodied by a rigid robot asset and the :ref:`habitat_sim.physics.VelocityControl` structure is used to control the robot's actions.
+
+.. include:: ../../examples/tutorials/rigid_object_tutorial.py
+    :code: py
+    :start-after: # [embodied_agent]
+    :end-before: # [/embodied_agent]
+
+.. image:: images/rigid-object-tutorial-images/robot_control.gif
+    :width: 20em
```

Comment:
This video was soo pretty!! This is awesome!

One largely minor request, which can be ignored because it's covered in other tutorials -- can we also show the depth frame (so make the sensor rgba and depth)? But as I said, it's not critical, so feel free to ship without it.

Reply:
No problem, easy addition. 👍